modules/developer_manual/pages/app/advanced/storage-backend.adoc

```diff
@@ -1,6 +1,6 @@
 = Storage Backends
 
-[[external-storage-backends]]
+[[storage-backend-external-storage-backends]]
```

Comment:
``storage-backend-external-storage-backends`` --> ``storage-backend_external-storage-backends``
pattern is name undersore topic
example represents for all of them

Reply:
Right you are.